CPP17.md

```diff
@@ -423,15 +423,15 @@ class Proxy {
 public:
   Proxy(Callable c): c(c) {}
   template <class... Args>
-  decltype(auto) operator()(Args&&... args) {
+  auto operator()(Args&&... args) {
     // ...
     return std::invoke(c, std::forward<Args>(args)...);
   }
 };
 auto add = [](int x, int y) {
```

Comment:
Since we're improving this section can you add the const qualifier to this for me?